src/e2e/global-setup.ts

```diff
@@ -30,4 +34,35 @@ async function globalSetup() {
   await browser.close();
 }
 
+async function maybeStartWebServer(): Promise<void> {
+  if (process.env.E2E_START_LOCAL_SERVER === "true") {
+    console.log("[e2e] - Starting the local web server...");
+
+    // define port and timeout
```

Comment:
Ah cool, in that case&hellip;

```suggestion
    // define timeout
```

Although I could also argue that it's fairly self explanatory at this point and the comment isn't useful and can be removed entirely (unless you want to clarify that it's a server startup/initialization timeout).